Framework/Algorithms/src/FilterEvents.cpp

```diff
@@ -477,17 +484,27 @@ void FilterEvents::processAlgorithmProperties() {
       }
     }
   } // END-IF: m_isSplitterRelativeTime
+
+  //-------------------------------------------------------------------------
+  // Deprecated properties
+  //-------------------------------------------------------------------------
+  bool splitSampleLogs = getProperty("SplitSampleLogs");
+  if (splitSampleLogs == false)
+    g_log.warning() << "Option SplitSampleLogs is deprecated. Logs will be split\n";
```

Comment:
Can we add a comment/word about this property being ignored?

Reply:
What is there missing that the log warning "explanation" is not addressing?

Reply:
I think `"Option SplitSampleLogs is deprecated and ignored. Logs will be split\n"` is that little bit more explicit.